evaluator/autoware_planning_evaluator/config/planning_evaluator.param.yaml

```diff
@@ -13,6 +13,7 @@
       - jerk
       - lateral_deviation
       - yaw_deviation
+      - nearest_pose_deviation
```

Comment:
```suggestion
      - nearest_pose_lateral_deviation
```

Reply:
After consulting with them, we came up with the variable/function name `lateral_tajectory_displacement`!